src/nri/sampler.go

```diff
@@ -72,6 +70,15 @@ func (cs *ContainerSampler) SampleAll(ctx context.Context, i *integration.Integr
 	}
 
 	for _, container := range containers {
+		metrics, err := cs.metrics.Process(container.ID)
+		if err != nil {
+			log.Error("error fetching metrics for container %v: %v", container.ID, err)
+			if _, ok := err.(*biz.ErrExitedContainerExpired); ok {
+				log.Debug("skipping exited container older than TTL")
```

Comment:
I think `Info` might be a better fit, as this message could be useful for users and not just developers.
```suggestion
				log.Info("skipping exited container older than TTL")
```